equinity-historical-data/src/test/resources/Crm14MockFile_5001817.txt

```diff
@@ -1755,11 +1755,13 @@
     "Cant_present_own_case": true,
     "Subformfundingdecision_1-Means_test_result_magsorcfs": "",
     "Charges_brought": {
-        "row": {
+            "row": [
```

Comment:
Do we still have a test file for single charge,  same as with no field?  I don't see the updated test cases

Reply:
added additional test case for single charge